posthog/async_migrations/utils.py

```diff
@@ -52,6 +52,14 @@ def process_error(migration_instance: AsyncMigration, error: Optional[str]):
     attempt_migration_rollback(migration_instance)
 
 
+def is_migration_resumable(migration_instance: AsyncMigration):
```

Comment:
name is slightly misleading as the migration isn't what's resumable but rather the op

Reply:
see the error message

Reply:
i know that - I just meant the name

Reply:
Maybe `can_resume_migration`? 

Indicates the same thing without ambiguity as to the general "type" of the migration